sdk/keyvault/Azure.Security.KeyVault.Keys/tests/SessionRecords/CryptographyClientLiveTests/WrapUnwrapRoundTrip(A128KW).json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Nit: These JSON files may not be needed now since their parameter values are excluded.